docs/search-v2.md

```diff
@@ -143,3 +143,206 @@ sleep(randint(1, 10))
 ```
 
 to `kitsune.search.v2.views.simple_search`.
+
+### Synonyms
+
+The `elasticsearch/dictionaries/synonyms` path contains a text file for each of our search-enabled locales,
+where synonyms are in the
+[Solr format](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/analysis-synonym-graph-tokenfilter.html#_solr_synonyms_2).
```

Comment:
I am curious as to why you chose synonym_graph and not synonym for token filter.

Reply:
discussed in meeting: synonym_graph is recommended upstream